pallets/identity/src/benchmarking.rs

```diff
@@ -15,56 +15,92 @@
 };
 use sp_std::prelude::*;
 
 const SEED: u32 = 0;
 
-fn setup_investor_uniqueness_claim<T: Trait + TestUtilsFn<AccountIdOf<T>>>(
+fn setup_investor_uniqueness_claim_common<T, P, IF, CF, SF, PF>(
     name: &'static str,
-) -> (User<T>, Claim, InvestorZKProofData) {
-    let mut user = UserBuilder::<T>::default().build(name);
-
-    let did = IdentityId::from([
-        152u8, 25, 31, 70, 229, 131, 2, 22, 68, 84, 54, 151, 136, 3, 105, 122, 94, 58, 182, 27, 30,
-        137, 81, 212, 254, 154, 230, 123, 171, 97, 74, 95,
-    ]);
-    Module::<T>::link_did(user.account(), did);
-    user.did = Some(did.clone());
-
-    let cdd_id = CddId::from([
-        102u8, 210, 32, 212, 213, 80, 255, 99, 142, 30, 202, 20, 220, 131, 109, 106, 137, 12, 137,
-        191, 123, 156, 212, 20, 215, 87, 23, 42, 84, 181, 128, 73,
-    ]);
-    let cdd_claim = Claim::CustomerDueDiligence(cdd_id);
-    Module::<T>::base_add_claim(did, cdd_claim, did, Some(666u32.into()));
-
-    let scope = Scope::Custom([228u8, 152, 116, 104, 5, 8, 30, 188, 143, 185, 10, 208].to_vec());
-    let scope_did = IdentityId::from([
-        2u8, 72, 20, 154, 7, 96, 116, 105, 155, 74, 227, 252, 172, 18, 200, 203, 137, 107, 200,
-        210, 194, 71, 250, 41, 108, 172, 100, 107, 223, 114, 182, 101,
-    ]);
-    let conf_scope_claim = Claim::InvestorUniqueness(scope, scope_did, cdd_id);
-
-    let inv_proof = InvestorZKProofData(
-        schnorrkel::Signature::from_bytes(&[
-            216u8, 224, 57, 254, 200, 45, 150, 202, 12, 108, 226, 233, 148, 213, 237, 7, 35, 150,
-            142, 18, 127, 146, 162, 19, 161, 164, 95, 67, 181, 100, 156, 25, 201, 210, 209, 165,
-            182, 74, 184, 145, 230, 255, 215, 144, 223, 100, 100, 147, 226, 58, 142, 92, 103, 153,
-            153, 204, 123, 120, 133, 113, 218, 51, 208, 132,
-        ])
-        .unwrap(),
-    );
-
-    (user, conf_scope_claim, inv_proof)
+    make_investor_uid: IF,
+    make_cdd_id: CF,
+    make_scope_id: SF,
+    make_proof: PF,
+) -> (User<T>, Claim, P)
+where
+    T: Trait + TestUtilsFn<AccountIdOf<T>>,
+    IF: Fn(&[u8]) -> InvestorUid,
+    CF: Fn(IdentityId, InvestorUid) -> CddId,
+    SF: Fn(&[u8], &InvestorUid) -> IdentityId,
+    PF: Fn(&IdentityId, &InvestorUid, &Ticker) -> P,
+{
+    let user = UserBuilder::<T>::default().generate_did().build(name);
+
+    // Create CDD and add it to `user`.
+    let did = user.did();
+    let investor_uid = make_investor_uid(did.as_bytes());
+    let cdd_id = make_cdd_id(did, investor_uid.clone());
+    let cdd_claim = Claim::CustomerDueDiligence(cdd_id.clone());
+    Module::<T>::base_add_claim(did, cdd_claim, GC_DID, None);
+
+    // Create the scope.
+    let ticker = Ticker::default();
+    let scope_id = make_scope_id(&ticker.as_slice(), &investor_uid);
+
+    // Claim
+    let claim = Claim::InvestorUniqueness(Scope::Ticker(ticker), scope_id, cdd_id);
+
+    // Proof
+    let proof = make_proof(&did, &investor_uid, &ticker);
+
+    (user, claim, proof)
```

Comment:
```suggestion
    let claim = Claim::InvestorUniqueness(Scope::Ticker(ticker), scope_id, cdd_id);
    let proof = make_proof(&did, &investor_uid, &ticker);
    (user, claim, proof)
```
The comments here add no value atop of the binding names, since they are well... identical.